### PR TITLE
Add fractional ephemeris rate observation request language to NEOExchange

### LINKED ISSUE #588: Add semi-nonsidereal scheduling
New semi-nonsidereal (half motion tracking) capabilities have been deployed.
Once this feature is available through the API, we should probably make this the default behavior of neoexchange.

As the current functionality does not have the capability of accounting for an object's drift on the chip, we will need to include multiple configurations within each request to reset the target center based on the maximum target speed within the window. 

We will need to decide if we will want to include the option to toggle this feature (and add a sidereal tracking option) to the advanced features tab of the schedule form.

### PR DESCRIPTION
**Pull request by jchate6**
Closes #588 
- Sets half rate to default
- adds rate selection to submission page (Additional options)
- splits configurations to ensure the target never leaves the FOV
- Split Instrument configurations so they progress in the expected way 

[NOTE: this currently does not split up instrument configurations with multiple exposures. For example, a 10 exposure block, split into 2 [5,5] exposure configurations, with the expected filter pattern of [V,R] will be unchanged (V,R,V,R,V,)(R,V,R,V,R), but filter pattern of [VV,RR] will not finish a partially completed pattern and result in (V,V,R,R,V)(R,R,V,V,R). This could cause issues for multi-filter blocks with per-filter exposure counts higher than the binned configs.]

Still TODO:
- [x] Add "rate" model field to `Block`
- [x] More Tests (MuSCAT, edge cases, spectroscopy)
- [x] deploy to Dev

### CODE REVIEW

File: neoexchange/core/migrations/0064_auto_20220128_2217.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def set_stellar_tracking(apps, schema_editor):
+    Block = apps.get_model('core', 'Block')
+    for star_block in Block.objects.exclude(calibsource=None):
+        star_block.tracking_rate = 0
+        star_block.save()
+
+
+class Migration(migrations.Migration):
```
Comment (talister) [inline neoexchange/core/migrations/0064_auto_20220128_2217.py:13]: Think it's worth putting a comment in this class to say that this is a Data Migration and include a link to the docs (https://docs.djangoproject.com/en/3.1/topics/migrations/#data-migrations)

**Review by talister — CHANGES_REQUESTED** [review]
Requested changes as discussed in the Zoom call of 2/2/2022. Need for more comprehensive functional tests around the "schedule form->schedule submit->block recording" sections noted but not to be implemented on this branch...

File: neoexchange/core/templates/core/schedule_confirm.html [hunk 2]
```diff
@@ -188,7 +188,7 @@ <h6 class="section-title">Calculated characteristics at {{ data.mid_time }}</h6>
                         </td>
                     </tr>
                     <tr id="id_speed_row">
-                        <td class="kv-key">Speed</td>
+                        <td class="kv-key">Speed / Trailing</td>
                         {% if data.trail_len > data.typical_seeing %}
                             <td class="kv-value">{{ data.speed|floatformat:2 }} "/min <i class="fa fa-arrow-right"></i>
                                 <table align="right"  style="margin: 0px"><tr><td></td><td class="kv-value"><div class="tooltip">
```
[the remark this answers is not in the document]
Comment (jchate6) [reply neoexchange/core/templates/core/schedule_confirm.html:194]: Done

**Comment by jchate6** [comment]
Aside from failing docker containers, I think this looks pretty good, if you want to give another look next week.
